packages/web-fetch/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import {reatomAsync} from '@reatom/async'
+import {Atom, Ctx, isAtom, throwReatomError} from '@reatom/core'
+
+export interface ReatomFetchOptions<O, I> {
```

Comment:
Let "query params" / "query string" / "query search" be "search"?
```suggestion
export interface ReatomFetchOptions<Payload, Params> {
```